src/utils/openCommentModal.js

```diff
@@ -0,0 +1,65 @@
+import { commentMovies, getComments } from '../api/apiInvolveComments.js';
+import { getSpecificMovie } from '../api/movieData.js';
+
+const modal = document.querySelector('#modal');
+const closeButton = document.querySelector('#modalCloseButton');
+const commentForm = document.getElementById('comment-form');
+const commentInput = document.getElementById('insights');
+const nameInput = document.getElementById('name');
+
+const onOpenModal = (id) => {
+  getSpecificMovie(id).then((item) => {
+    document.body.style.overflow = 'hidden';
+    const thumbnail = new Image();
+    thumbnail.src = item.image.original;
+    const modalTitle = document.getElementById('modalTitle');
+    const modalImage = document.getElementById('thumbnail');
+    const info1 = document.querySelector('.info1');
+    const info2 = document.querySelector('.info2');
+    const info3 = document.querySelector('.info3');
+    const info4 = document.querySelector('.info4');
+    const comment = document.getElementById('comments');
+    const commentTitle = document.querySelector('.comments-title');
+    modalImage.appendChild(thumbnail);
+    modalTitle.innerText = item?.name;
+    info1.innerHTML = `Language :${item.language}`;
+    info2.innerHTML = `Rating :${item.rating.average}`;
+    info3.innerHTML = `Status :${item.status}`;
+    info4.innerHTML = `Run Time :${item.averageRuntime} min`;
+
+    console.log(item);
```

Comment:
**It's Optional**
Kindly remove the console.log function

Reply:
thanks for your amazing review :heart: